tests/displays/cavity_display/frontend/test_fault_count_display.py

```diff
@@ -18,6 +18,15 @@ def window():
     yield FaultCountDisplay(lazy_fault_pvs=True)
 
 
+# Test to see if we make a list of strings with no duplicates
+def test_make_tlc_list(qtbot: QtBot, window):
+    test_result_tlc_list = window.make_tlc_list()
+    assert isinstance(test_result_tlc_list, list)
+    for tlc in test_result_tlc_list:
+        assert isinstance(tlc, str)
+    assert (len(test_result_tlc_list) == len(set(test_result_tlc_list))) == True
```

Comment:
side note this has a syntax error lol the `== True`

Reply:
Oh no, what's the syntax error?? I was trying to make a line that checks to make sure there are no duplicates in the list, but I don't actually know if that needed to be there or not...

Reply:
The `== True` is the syntax error, you just need `assert len(test_result_tlc_list) == len(set(test_result_tlc_list))`

Reply:
OHHHHH I see. Fixed!